cmd/log-validator/main.go

```diff
@@ -43,9 +44,16 @@ func lineValid(text string) error {
 	checksum := fields[5]
 	// Reconstruct just the message portion of the line
 	line := strings.Join(fields[6:], " ")
+
+	const errorPrefix = "invalid checksum"
```

Comment:
This is fine, but long-term, it would be better for all log lines to be prefixed with the file/line they come from, so this could be a much more accurate check. (I can think of a few reasons off-hand why a totally unrelated log line would contain this same string.)